src/commands.rs

```diff
@@ -8,7 +8,7 @@ use reqwest::blocking::Client as HttpClient;
 use serenity::{model::channel::Message, prelude::Context};
 use std::{collections::HashMap, sync::Arc};
 
-const PREFIX: &'static str = "?";
+pub(crate) const PREFIX: &'static str = "?";
```

Comment:
This should be a configuration at some point. Is this in scope for this PR?

Reply:
Out of scope, but I dont disagree entirely.